[
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/259467567",
    "html_url": "https://github.com/jupyter/notebook/issues/1881#issuecomment-259467567",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1881",
    "id": 259467567,
    "node_id": "MDEyOklzc3VlQ29tbWVudDI1OTQ2NzU2Nw==",
    "user": {
      "login": "takluyver",
      "id": 327925,
      "node_id": "MDQ6VXNlcjMyNzkyNQ==",
      "avatar_url": "https://avatars.githubusercontent.com/u/327925?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/takluyver",
      "html_url": "https://github.com/takluyver",
      "followers_url": "https://api.github.com/users/takluyver/followers",
      "following_url": "https://api.github.com/users/takluyver/following{/other_user}",
      "gists_url": "https://api.github.com/users/takluyver/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/takluyver/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/takluyver/subscriptions",
      "organizations_url": "https://api.github.com/users/takluyver/orgs",
      "repos_url": "https://api.github.com/users/takluyver/repos",
      "events_url": "https://api.github.com/users/takluyver/events{/privacy}",
      "received_events_url": "https://api.github.com/users/takluyver/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-11-09T17:05:25Z",
    "updated_at": "2016-11-09T17:05:25Z",
    "author_association": "MEMBER",
    "body": "Pinging @tacaswell\n",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/259879139",
    "html_url": "https://github.com/jupyter/notebook/issues/1881#issuecomment-259879139",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1881",
    "id": 259879139,
    "node_id": "MDEyOklzc3VlQ29tbWVudDI1OTg3OTEzOQ==",
    "user": {
      "login": "tacaswell",
      "id": 199813,
      "node_id": "MDQ6VXNlcjE5OTgxMw==",
      "avatar_url": "https://avatars.githubusercontent.com/u/199813?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/tacaswell",
      "html_url": "https://github.com/tacaswell",
      "followers_url": "https://api.github.com/users/tacaswell/followers",
      "following_url": "https://api.github.com/users/tacaswell/following{/other_user}",
      "gists_url": "https://api.github.com/users/tacaswell/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/tacaswell/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/tacaswell/subscriptions",
      "organizations_url": "https://api.github.com/users/tacaswell/orgs",
      "repos_url": "https://api.github.com/users/tacaswell/repos",
      "events_url": "https://api.github.com/users/tacaswell/events{/privacy}",
      "received_events_url": "https://api.github.com/users/tacaswell/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-11-11T04:33:56Z",
    "updated_at": "2016-11-11T04:33:56Z",
    "author_association": "NONE",
    "body": "I have a good guess as to what is going wrong and it is due to the auto-redraw on stale logic.\n\npost-execute event checks if any figures are stale -> figure is stale -> triggers redraw -> redraw fails -> the figure is still stale ->  post-execute event checks if any figures are stale -> ...\n",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/260007395",
    "html_url": "https://github.com/jupyter/notebook/issues/1881#issuecomment-260007395",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1881",
    "id": 260007395,
    "node_id": "MDEyOklzc3VlQ29tbWVudDI2MDAwNzM5NQ==",
    "user": {
      "login": "takluyver",
      "id": 327925,
      "node_id": "MDQ6VXNlcjMyNzkyNQ==",
      "avatar_url": "https://avatars.githubusercontent.com/u/327925?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/takluyver",
      "html_url": "https://github.com/takluyver",
      "followers_url": "https://api.github.com/users/takluyver/followers",
      "following_url": "https://api.github.com/users/takluyver/following{/other_user}",
      "gists_url": "https://api.github.com/users/takluyver/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/takluyver/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/takluyver/subscriptions",
      "organizations_url": "https://api.github.com/users/takluyver/orgs",
      "repos_url": "https://api.github.com/users/takluyver/repos",
      "events_url": "https://api.github.com/users/takluyver/events{/privacy}",
      "received_events_url": "https://api.github.com/users/takluyver/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-11-11T17:18:34Z",
    "updated_at": "2016-11-11T17:18:34Z",
    "author_association": "MEMBER",
    "body": "So that would be a mpl issue? Feel free to use this thread to work out the details, and let us know if we can do anything on our side.\n",
    "performed_via_github_app": null
  },
  {
    "url": "https://api.github.com/repos/jupyter/notebook/issues/comments/260842087",
    "html_url": "https://github.com/jupyter/notebook/issues/1881#issuecomment-260842087",
    "issue_url": "https://api.github.com/repos/jupyter/notebook/issues/1881",
    "id": 260842087,
    "node_id": "MDEyOklzc3VlQ29tbWVudDI2MDg0MjA4Nw==",
    "user": {
      "login": "tacaswell",
      "id": 199813,
      "node_id": "MDQ6VXNlcjE5OTgxMw==",
      "avatar_url": "https://avatars.githubusercontent.com/u/199813?v=4",
      "gravatar_id": "",
      "url": "https://api.github.com/users/tacaswell",
      "html_url": "https://github.com/tacaswell",
      "followers_url": "https://api.github.com/users/tacaswell/followers",
      "following_url": "https://api.github.com/users/tacaswell/following{/other_user}",
      "gists_url": "https://api.github.com/users/tacaswell/gists{/gist_id}",
      "starred_url": "https://api.github.com/users/tacaswell/starred{/owner}{/repo}",
      "subscriptions_url": "https://api.github.com/users/tacaswell/subscriptions",
      "organizations_url": "https://api.github.com/users/tacaswell/orgs",
      "repos_url": "https://api.github.com/users/tacaswell/repos",
      "events_url": "https://api.github.com/users/tacaswell/events{/privacy}",
      "received_events_url": "https://api.github.com/users/tacaswell/received_events",
      "type": "User",
      "site_admin": false
    },
    "created_at": "2016-11-16T03:14:41Z",
    "updated_at": "2016-11-16T03:14:41Z",
    "author_association": "NONE",
    "body": "What are the conditions when 'post_execute' hook is called?  \n\nFixing the infinite loop is relatively strait forward (just mark the figure as not stale, even if drawing failed), but I am not fully convinced _correct_.\n\nGetting the exception to propagate across is going to be more difficult an the draw is run via a callback from the js side via a comm we have set up (this is the same reason of the other issues where tracebacks and `print` calls from mouse / keyboard events do not make it back out to the notebook.  These issues _should_ be addressed by `ipympl`.\n",
    "performed_via_github_app": null
  }
]
